developer/src/vscode-plugin/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@keymanapp/developer-vscode",
+  "displayName": "Keyman Developer for VSCode",
+  "description": "Plugin for compiling Keyman projects in VSCode",
+  "engines": {
+    "vscode": "^1.93.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/keymanapp/keyman.git"
+  },
+  "categories": [
+    "Other"
+  ],
+  "activationEvents": [],
+  "main": "./out/extension.js",
+  "contributes": {
+  },
+  "scripts": {
+    "vscode:prepublish": "npm run compile",
+    "compile": "tsc -p ./",
+    "watch": "tsc -watch -p ./",
```

Comment:
I notice some of the comments from previous PR haven't yet been applied here, is that intentional?

Reply:
reviewing.